ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/part/editor/EditorPartStackView.java

```diff
@@ -48,28 +51,63 @@
 
   interface PartStackUiBinder extends UiBinder<Widget, EditorPartStackView> {}
 
+  /** Listener to handle clicking on Add tab button. */
+  interface AddTabButtonClickListener {
+
+    /**
+     * Is called when Add tab button clicked.
```

Comment:
`Is called when Add tab button clicked?`
OR
`Called when Add tab button clicked.`